feconf.py

```diff
@@ -348,6 +349,14 @@ def get_empty_ratings():
     },
 }
 
+# Ids of the allowed classification algorithm classes.
+ALLOWED_CLASSIFIER_CLASSES = [{
```

Comment:
Just like 'ALLOWED_INTERACTION_CATEGORIES'. But will change it if it doesn't make sense.